server/controllers/group_controller.js

```diff
@@ -107,9 +106,10 @@ Input:
 Output: Rejects if groupId is not found;
 otherwise resolves to array of article objects that are in the group.
 */
-export const getGroupArticles = (groupId) => {
+// TODO: we don't actually have a router function for this quite yet
+export const getGroupArticles = (groupId, pagination_options) => {
```

Comment:
We also need to implement a "getGroupArticles" endpoint I think because it seems like we never use this? Confusing because not totally sure what front end endpoint Feed View is hitting right now to populate if not this function?